candies/cli/cli.py

```diff
@@ -13,11 +14,14 @@ class CLI:
         main: A main command to execute.
         parser: A parser to parse command line arguments with
             (`candies.cli.parsers.StandardParser` if not specified).
+        parsers: A dictionary that contains parsers for custom types.
+            Maps a custom to a parser itself.
```

Comment:
`Maps a custom to a parser itself` -- seems like a word is missing.

Reply:
Yes, will fix that!